azure-devops/cmake-configure-build.yml

```diff
@@ -38,7 +38,11 @@ steps:
     -DCMAKE_CXX_COMPILER=cl ^
     -DCMAKE_BUILD_TYPE=Release ^
     -DLIT_FLAGS=$(litFlags) ^
-    -DCMAKE_CXX_FLAGS=/analyze:autolog- ^
+    -DCMAKE_CXX_FLAGS="/showIncludes /analyze:autolog-" ^

```

Comment:
Is `/showIncludes` a relic of development? I am concerned about the amount of log spam it could generate.